I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 279, in script
    python:
ScriptError: could not find label 'start'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/home/phong/Downloads/renpy-8.3.3-sdk/renpy/bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "/home/phong/Downloads/renpy-8.3.3-sdk/renpy/main.py", line 652, in main
    run(restart)
  File "/home/phong/Downloads/renpy-8.3.3-sdk/renpy/main.py", line 148, in run
    renpy.execution.run_context(True)
  File "/home/phong/Downloads/renpy-8.3.3-sdk/renpy/execution.py", line 958, in run_context
    context.run()
  File "renpy/common/00start.rpy", line 279, in script
    python:
  File "/home/phong/Downloads/renpy-8.3.3-sdk/renpy/script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'start'.

Linux-6.12.9-arch1-1-x86_64-with-glibc2.40 x86_64
Ren'Py 8.3.3.24111502
Moral - Abyss's Lullaby 1.0
Sat Jan 18 18:54:25 2025
